### PR TITLE
Add `console.assert`

### PR DESCRIPTION
**Pull request by g-plane**
I'm not sure that if the assertion failed an error should be thrown.

### CODE REVIEW

**Comment by g-plane** [comment]
@qti3e Can you review again please?

File: testdata/013_console_assert.ts.out [hunk 1]
```diff
@@ -0,0 +1 @@
+You will see me.
```
Comment (qti3e) [inline testdata/013_console_assert.ts.out]: it will be: "Assertion failed: You will see me. "

Comment (qti3e) [reply testdata/013_console_assert.ts.out]: you can do:
./deno 013_console_assert.ts > 013_console_assert.ts.out

Comment (g-plane) [reply testdata/013_console_assert.ts.out]: Sorry, that is my fault. I will update.

Comment (qti3e) [reply testdata/013_console_assert.ts.out]: @g-plane No problem : )

Comment (qti3e) [reply testdata/013_console_assert.ts.out]: you should put one blank space ` ` at the end of your `.out` file.
see: stringifyArgs:
https://github.com/ry/deno/blob/7b5fbc71970056515c41f6983fc8e1f8ee443d11/globals.ts#L39-L49

**Comment by g-plane** [comment]
CI passed.

**Comment by yorkie** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/console/assert Do we need follow the naming and full implementation? This patch seems not do like this.

**Comment by g-plane** [comment]
@yorkie Can you give some examples?

**Comment by ry** [comment]
Thanks! I'd rather this was tested in tests.ts. The `testdata` tests start up a whole process and run rather slowly.

**Comment by g-plane** [comment]
@ry If we put it into `tests.ts`, how to check the console output?

**Comment by ry** [comment]
@g-plane It should throw - sorry didn't catch that it didn't before - and you can use a try-catch to test it.

**Comment by g-plane** [comment]
@ry Updated and CI passed.

**Comment by ry** [comment]
@g-plane Thanks!

**Pull request merged**